Makefile

```diff
@@ -71,7 +71,7 @@ vet: ## Run go vet against code
 	go vet ./pkg/... ./cmd/...
 
 .PHONY: generate
-generate: clientset ## Generate code
+generate: clientset gazelle ## Generate code
```

Comment:
Technically the old way guaranteed that gazelle was run last, but in practice we just can't run this makefile in parallel.